### PR TITLE
add docs for publishing to github pages manually

### PR DESCRIPTION
**Pull request by bluejekyll**
I had some trouble with the way that mdbook cleans the target directory, this documents how I worked around that. I'll try and come up with a patch for clean, that leaves `.git` in tact.

### CODE REVIEW

File: book-example/src/continuous-integration.md [hunk 1]
```diff
@@ -54,3 +54,35 @@ deploy:
 ```
 
 That's it!
+
+### Deploying to GitHub Pages manually
+
+If you're CI doesn't support a GitHub pages, or you're deploying somewhere else
```
Comment (Dylan-DPC) [inline book-example/src/continuous-integration.md]:
```suggestion
If your CI doesn't support a GitHub pages, or you're deploying somewhere else
```

File: book-example/src/continuous-integration.md [hunk 2]
```diff
@@ -54,3 +54,35 @@ deploy:
...
+with GitHub pages like integration, *note: you may want to use different tmp dirs*:
```
Comment (Dylan-DPC) [inline book-example/src/continuous-integration.md]:
```suggestion
with integrations such as Github Pages: 
 *note: you may want to use different tmp dirs*:
```

**Comment by bluejekyll** [comment]
@Dylan-DPC , missed your comments. I accepted all of them.

**Comment by Dylan-DPC** [comment]
Thanks

**Pull request merged**